explorer/src/utils/stringUtils.ts

```diff
@@ -13,3 +13,7 @@ export const cropString = (query: string, step = 4, headFromIndex = 0) => {
 };
 
 export const removeCommas = (str: string) => str.replace(/,/g, "");
+
+export const isValidId = (id: string, regex: RegExp) => {
```

Comment:
Creating a separate function just for testing for a regex seems to be redundant.